src/main/java/com/example/history/HistoryController.java

```diff
@@ -0,0 +1,30 @@
+package com.example.history;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class HistoryController {
+    private HistoryMapper historyMapper;
+    public HistoryController(HistoryMapper historyMapper){
+        this.historyMapper = historyMapper;
+    }
+    @GetMapping("/history")
```

Comment:
一般的にRESTではリソース名の複数形をパスに使います。
```suggestion
    }
    @GetMapping("/histories")
```